### PR TITLE
Documentation and Example for pv-scripts

### PR DESCRIPTION
**Pull request by deveth0**

### CODE REVIEW

**Comment by deveth0** [comment]
Please have a look into the MR (especially the code-style in the example) and provide a review.

File: examples/react-tsx/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "dev": "pv-scripts dev",
+    "prod": "pv-scripts prod"
+  },
+  "author": "Alexander Muthmann <amuthmann@pro-vision.de>",
+  "license": "ISC",
+  "devDependencies": {
+    "@pro-vision/pv-scripts": "^1.2.2",
+    "@types/react": "^16.9.2",
+    "@types/react-dom": "^16.9.0",
+    "copy-webpack-plugin": "^5.0.4",
```
Comment (friewerts) [inline examples/react-tsx/package.json]: this dependency is not needed anymore

File: examples/react-tsx/src/js/legacyIndex.tsx [hunk 2]
```diff
@@ -0,0 +1 @@
+import "./app"
```
Comment (friewerts) [inline examples/react-tsx/src/js/legacyIndex.tsx:3]: To showcase the legacy bundle. You could add a legacy polyfill (e.g. fetch) and use it.

File: examples/react-tsx/src/components/Hello.tsx [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import * as React from "react";
+
+export interface HelloProps { compiler: string; framework: string; }
```
Comment (friewerts) [inline examples/react-tsx/src/components/Hello.tsx]: you could also define the HelloProps as type here, as you are not using any extention by now. interface is also fine though.

Comment (friewerts) [reply examples/react-tsx/src/components/Hello.tsx]: you don't need to export the HelloProps. In this moment, they are just used in this file.

File: examples/react-tsx/src/js/app.tsx [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
```
Comment (friewerts) [inline examples/react-tsx/src/js/app.tsx]: You could use `import { render } from "react-dom";` to just import the render funktion

File: examples/react-tsx/src/components/Hello.tsx (hunk 3 above)
Comment (deveth0) [reply examples/react-tsx/src/components/Hello.tsx]: thx. done 79d87ad

File: examples/react-tsx/src/js/app.tsx (hunk 4 above)
Comment (deveth0) [reply examples/react-tsx/src/js/app.tsx]: thx. done 79d87ad

File: examples/react-tsx/package.json (hunk 1 above)
Comment (deveth0) [reply examples/react-tsx/package.json]: thx. done

File: examples/react-tsx/src/js/legacyIndex.tsx (hunk 2 above)
Comment (deveth0) [reply examples/react-tsx/src/js/legacyIndex.tsx:3]: thx. done 79d87ad

**Pull request merged**